src/k8s-engine/lima.ts

```diff
@@ -917,20 +955,48 @@ ${ commands.join('\n') }
   protected async installCustomLimaNetworkConfig() {
     const networkPath = path.join(paths.lima, '_config', 'networks.yaml');
 
-    try {
-      const data = yaml.parse(await fs.promises.readFile(networkPath, 'utf8'));
-      const runFile = data?.paths?.varRun ?? '';
+    let config: LimaNetworkConfiguration;
 
-      if (runFile.includes('/rancher-desktop')) {
-        // Assume if there's a paths.varRun setting mentioning "rancher-desktop" there's no need to replace it.
-        return;
+    try {
+      config = yaml.parse(await fs.promises.readFile(networkPath, 'utf8'));
+      if (config?.paths?.varRun !== NETWORKS_CONFIG.paths.varRun) {
+        console.log('Lima network configuration has unexpected contents; it will be replaced.');
```

Comment:
Maybe add the old contents to the log, so you have a (brief) chance to recover it?